README.md

```diff
@@ -209,7 +209,7 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_frontend.{n}.maxconn`: [optional]: Fix the maximum number of concurrent connections
 * `haproxy_frontend.{n}.logformat`: [optional]: Specifies the log format string to use for traffic logs (e.g. `'"%{+Q}o\ %t\ %s\ %{-Q}r"'`)
 * `haproxy_frontend.{n}.stick`: [optional]: Stick declarations
-* `haproxy_frontend.{n}.stick.{n}.table`: [required]: Configure the stickiness table for the current section (e.g. `type ip size 500k`)
+* `haproxy_frontend.{n}.stick.{n}.table`: [optional]: Configure the stickiness table for the current section (e.g. `type ip size 500k`)
```

Comment:
I don't see any changes below for this `required -> optional`. Is that correct?

Reply:
Apologies for the half-backed PR - Would you agree the best approach would be to make stick table optional for frontend, backend, and listen?